client/functions.lua

```diff
@@ -918,3 +946,65 @@ function QBCore.Functions.StartParticleOnEntity(dict, ptName, looped, entity, bo
     end
     return particleHandle
 end
+
+function QBCore.Functions.GetStreetNametAtCoords(coords)
+    local streetname1, streetname2  	= GetStreetNameAtCoord(coords.x, coords.y, coords.z)
```

Comment:
Weird space between the variables and the definer

Reply:
Modified :)